include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -474,16 +446,6 @@ struct __radix_sort_scan_submitter<_RadixLocalScanName, __internal::__optional_k
                                                        _CountT(0), __dpl_sycl::__plus<_CountT>{});
                 });
         });
-
-        // 2. Global scan: produces global offsets using local offsets
-        __scan_event = __exec.queue().submit([&](sycl::handler& __hdl) {
-            __hdl.depends_on(__scan_event);
-            oneapi::dpl::__ranges::__require_access(__hdl, __count_rng);
-            __hdl.parallel_for<_RadixGlobalScanName...>(
```

Comment:
Could you also remove these unused entities?
- _RadixGlobalScanName
- __radix_sort_scan_kernel_2

Reply:
You may consider renaming '__radix_sort_scan_kernel_1' to '__radix_sort_scan_kernel' as well.